classes/item.rb

```diff
@@ -0,0 +1,21 @@
+require 'date'
+
+class Item
+  attr_accessor :id, :publish_date, :archived, :genre, :author, :label, :source
+
+  def initialize(publish_date)
+    @id = Random.rand(1..100)
+    @publish_date = publish_date
+    @archived = false
+  end
```

Comment:
Hello @techEmprez  @IamYunusAli 
i think 
other proprieties of an item such as

- `@gener `
- `@author`
- `@source`
- `@lable`

 Those should be **initialized** but **not set** in the initialized method instead 4 methods should be added to the item class 
 which are:
add_source(name)
 add_gener(name)
add_author(firstName,lastName)
add_lable(title,color)
and those then should be called when creating the book 
and their arguments are going to be ` user_input`